scripts/docs/parse_assets.py

```diff
@@ -31,34 +31,85 @@ def parse_assets(input_dirs: List[Path],
     """
     asset_count = 0
 
-    references = defaultdict(list)
+    references = {}
```

Comment:
You could also use:
```suggestion
    references = defaultdict(lambda: defaultdict(list))
```
See https://stackoverflow.com/a/27809959